.vscode/launch.json

```diff
@@ -284,56 +284,28 @@
             "url": "http://localhost:1344",
             "preLaunchTask": "Node Render Graph Editor Serve for core (Dev)"
         },
-        {
-            "type": "msedge",
-            "request": "launch",
-            "name": "VSM development (Edge)",
-            "url": "http://localhost:1342",
-            "preLaunchTask": "VSM Serve (Dev)"
-        },
-        {
-            "type": "chrome",
-            "request": "launch",
-            "name": "VSM development (Chrome)",
-            "url": "http://localhost:1342",
-            "preLaunchTask": "VSM Serve (Dev)"
-        },
-        {
-            "type": "msedge",
-            "request": "launch",
-            "name": "Launch VSM (Edge)",
-            "url": "http://localhost:1342",
-            "preLaunchTask": "VSM Serve for core (Dev)"
-        },
-        {
-            "type": "chrome",
-            "request": "launch",
-            "name": "Launch VSM (Chrome)",
-            "url": "http://localhost:1342",
-            "preLaunchTask": "VSM Serve for core (Dev)"
-        },
-        {
-            /*
-            Launch the playground to be use to test changes in dev packages
-            */
-            "type": "msedge",
-            "request": "launch",
-            "name": "Run and Watch Dev host (ES6-Dev)",
-            "url": "http://localhost:1338",
-            "outFiles": ["!**/node_modules/**"],
-            "preLaunchTask": "Run and Watch Dev host (Dev)"
-        },
-        {
-            /*
-            Launch the playground to be use to test changes in dev packages
-            */
-            "type": "msedge",
-            "request": "launch",
-            "name": "Run and Watch Dev host - SSL (ES6-Dev)",
-            "url": "https://localhost:1338",
-            "outFiles": ["!**/node_modules/**"],
-            "preLaunchTask": "Run and Watch Dev host (Dev)"
-        },
+        // {

```

Comment:
should we not just remove them ?

Reply:
Of course! Actually meant to do it here. I'll take care of that later